### PR TITLE
Initial pass at restructuring styling content for tailwind and Sass...

### PR DESCRIPTION
**Pull request by kristoferjoseph**
users.

[WIP]
Wanted to get this up so there is a place for the other content to be added.
Still needs additions like "badass" and "FOUCing" added to the dictionary so the spellcheck is going to fail.

### CODE REVIEW

File: app/docs/md/learn/concepts/styling/index.md [hunk 1]
```diff
@@ -2,64 +2,45 @@
 title: Styling
 ---
 
-An important part of both user experience and developer experience is applying styles to your app.
-There are a lot of solutions for approaching CSS, but very few that are designed for both the user and the developer in mind.
+Since Enhance is standards based you are free to style however you like. You can add a stylesheet to the public folder and start cowboying styles like an absolute badass or bring whatever tooling you like as long as it can output a static CSS file to the public folder of your app you are good to go.
```
Comment (macdonst) [inline app/docs/md/learn/concepts/styling/index.md:6]: I'd re-order the below to be:

1. Enhance Styles
2. Plain old CSS like we used to do, but better
3. Sass
4. Tailwind

File: app/docs/md/learn/concepts/styling/css.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        }
+      </style>
+    </head>
+`
+}
+```
+
+</doc-code>
+
+The benefits of adding styles to a style tag in the head is that the styles will be immediately available when the page loads which will help avoid FOUCing your users.
```
Comment (macdonst) [inline app/docs/md/learn/concepts/styling/css.md:43]: Maybe say the below to remove one repetition of style(s) in the sentence?

"The benefits of adding a style tag in the head is that the styles will be immediately available…

File: app/docs/md/learn/concepts/styling/using-tailwind.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+title: Using Tailwind
+---
+
```
Comment (macdonst) [inline app/docs/md/learn/concepts/styling/using-tailwind.md:4]: Left my comments on the below in Cole's google doc.

File: app/docs/md/learn/concepts/styling/index.md (hunk 1 above)
Comment (kristoferjoseph) [reply app/docs/md/learn/concepts/styling/index.md:6]: People want CSS first. Then we can ease them in.

File: app/docs/md/learn/concepts/styling/css.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+Add a `styles.css` file to the public folder.
+
+```
+app
+├── public ............ static assets
+│   └── styles.css
+└── head.mjs .......... custom <head> component
+```
+
+Then open up the `head.mjs` file and link to your new `styles.css` file at `/_public/styles.css`. Obviously you can put your CSS file in any nested folder you like in the public folder and name the file whatever you like, just remember update the path in your link tag.
```
Comment (colepeters) [inline app/docs/md/learn/concepts/styling/css.md:59]:
```suggestion
Then open up the `head.mjs` file and link to your new `styles.css` file at `/_public/styles.css`. Obviously you can put your CSS file in any nested folder you like in the public folder and name the file whatever you like, just remember to update the path in your link tag.
```

File: app/docs/md/learn/concepts/styling/css.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+<doc-callout level="none" mark="🔔">
+
+  `/_public` is a special path that will automatically forward to a fingerprinted version of your static assets.
```
Comment (colepeters) [inline app/docs/md/learn/concepts/styling/css.md:63]:
```suggestion
  [`/_public`](/docs/learn/starter-project/public) is a special path that will automatically forward to a fingerprinted version of your static assets.
```

File: app/docs/md/learn/concepts/styling/index.md [hunk 6]
```diff
@@ -2,64 +2,45 @@
...
-**[Read more about modular scales→](/docs/learn/concepts/styling/modular-scales)**
+**[Read more about using Sass →](/docs/learn/concepts/styling/using-sass)**
 
 </doc-callout>
 
-## Element styles
+## Enhance styles
 
-Certain styles would not be appropriate as a utility class. For these cases, you can write specific styles using the `<style>` tag in your custom element. These styles will be scoped to the element they’re declared in (and its children), and can take advantage of Enhance Styles’ generated custom properties.
+Enhance is a "batteries included" full-stack framework so we've preconfigured a styling system that is the culmination of decades of CSS research and development into a "best of all worlds" approach to styling. It isn't what all the cool kids are using... yet, but we feel that if you give it a chance the benefits of our **deep** styling experience will become apparent.
```
Comment (colepeters) [inline app/docs/md/learn/concepts/styling/index.md:40]: This sentence is a bit long and complex; suggested edit to make it more concise without losing the information:

```suggestion
Enhance is a "batteries included" full-stack framework and comes with a preconfigured (and customizable) "best of all worlds" styling system, comprising decades of CSS research and development. It isn't what all the cool kids are using — yet! — but we feel that if you give it a chance, the benefits of our **deep** styling experience will become apparent.
```

**Review by colepeters — COMMENTED** [review]
Love these updates! Just a few suggestions for grammar and clarity

File: app/docs/nav-data.mjs [hunk 7]
```diff
@@ -73,11 +73,24 @@ export const data = [
...
             path: '/docs/learn/concepts/styling/',
             items: [
-              'utility-classes',
-              'logical-properties',
-              'customization',
-              'modular-scales',
-              'element-styles',
+              {
+                slug: 'CSS',
+                description: 'Modern CSS is pretty good',
```
Comment (colepeters) [inline app/docs/nav-data.mjs:78]: I'd even take this further! E.g. 'Modern CSS is amazing' :D

**Review by colepeters — APPROVED** [review]

**Comment by kristoferjoseph** [comment]
Had to add "Cribsheet" and "cribsheet" 😅

**Pull request merged**